src/main/java/com/transportcompany/transportcompany/controllers/OrderController.java

```diff
@@ -0,0 +1,43 @@
+package com.transportcompany.transportcompany.controllers;
+
+import com.transportcompany.transportcompany.models.dtos.OrderDTO;
+import com.transportcompany.transportcompany.models.entities.Order;
+import com.transportcompany.transportcompany.services.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/orders")
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    @PostMapping
+    public ResponseEntity<OrderDTO> createOrder(@RequestBody OrderDTO orderDTO, Object OrderDTO) {
+        return ResponseEntity.ok(orderService.createOrder(orderDTO));
+    }
+
+    @GetMapping("api/order")
```

Comment:
If you have defined on the class level `/api/orders`, you don t need to do it again on the method level. More than that, the name of the endpoint is correct? If we want to retrieve orders (plural), the resource in the URL should be `/api/orders` not order.